.eleventy.js

```diff
@@ -14,29 +16,55 @@ const taxonomy = require('#/taxonomy');
 const time = require('#/time');
 const type = require('#/type');
 const utils = require('#/utils');
-const images = require('#/images');
 
 // eleventy image plugin
-function imageShortcode(src, cls, alt, sizes, widths) {
-  let options = {
-    widths: widths,
-    formats: ['jpeg'],
+const imageShortcode = (src, alt, attrs, sizes, getUrl) => {
+  const imgConfig = taxonomy.fromTaxonomy('img');
+  const imgSizes =
+    sizes && imgConfig.sizes[sizes]
+      ? imgConfig.sizes[sizes]
+      : sizes || imgConfig.sizes.default;
+
+  const options = {
+    widths: imgConfig.widths,
+    formats: imgConfig.formats,
+    // test path and directory that will be changed in this PR
+    urlPath: '/img/',
+    outputDir: './_site/img/',
+    // eslint-disable-next-line
+    filenameFormat: function (id, src, width, format, options) {
+      const extension = path.extname(src);
+      const name = path.basename(src, extension);
+
+      return `${name}-${width}w.${format}`;
```

Comment:
Typo?

```suggestion
      return `${name}-${width}.${format}`;
```

Reply:
I assumed that meant "width"? But I don't know that it matters one way or the other.